WPMedia/engine-theme-sdk
# PEN-1237/update alc documentation

## Description
_This PR updates the user facing ALC documentation for the Engine SDK. Specifically:_

- The documentation meant for SDK devs, not client devs utilizing the SDK, has been moved from storybook to the readme. Some of this documentation (for example, how to publish), was in both places, so it was consolidated in the process.
- The alt text for images was referencing a client, so their name has been removed and accurate alt text used instead
- The arc logo, while part of the SDK, is not meant for client use. The documentation has been kept, but it's been moved to an internal folder where it will not be build as part of the ALC process.

## Jira Ticket
- [PEN-1237](https://arcpublishing.atlassian.net/browse/PEN-1237)

## Test Steps
- Run `npm run storybook`
- Confirm listed changes have been made.
